subsys/bluetooth/Kconfig.iso

```diff
@@ -113,6 +115,8 @@ config BT_ISO_RX_MTU
 	range 23 4095
 	help
 	  Maximum MTU for Isochronous channels RX buffers.
+	  This is the actual data payload. It doesn't include the optional
+	  HCI ISO Data packet fields (ie. `struct bt_hci_iso_ts_data_hdr`)
```

Comment:
```suggestion
	  HCI ISO Data packet fields (e.g. `struct bt_hci_iso_ts_data_hdr`)
```

Reply:
Thank you, I fixed BT_ISO_RX_MTU description